boards.txt

```diff
@@ -11632,6 +11731,148 @@ heltec_wifi_lora_32_V2.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_wifi_lora_V3.name=Heltec WiFi LoRa 32(V3) / Wireless shell(V3) / Wireless stick lite (V3)
+heltec_wifi_lora_V3.vid.0=0x303a
+heltec_wifi_lora_V3.pid.0=0x1001
+
+heltec_wifi_lora_V3.bootloader.tool=esptool_py
+heltec_wifi_lora_V3.bootloader.tool.default=esptool_py
+
+heltec_wifi_lora_V3.upload.tool=esptool_py
+heltec_wifi_lora_V3.upload.tool.default=esptool_py
+heltec_wifi_lora_V3.upload.tool.network=esp_ota
+
+heltec_wifi_lora_V3.upload.maximum_size=1310720
```

Comment:
Same here. When partition is fixed to default_8MB, I can't see that upload.maximus_size corresponds to the partition default_8MB.

`esp32s3.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336`